packages/frontend/src/components/DashboardFilter/Filter.tsx

```diff
@@ -219,25 +212,27 @@ const Filter: FC<Props> = ({
             </Popover.Target>
 
             <Popover.Dropdown ml={5}>
-                <FilterConfiguration
-                    isCreatingNew={isCreatingNew}
-                    isEditMode={isEditMode}
-                    isTemporary={isTemporary}
-                    field={field}
-                    fields={allFilterableFields || []}
-                    tiles={dashboardTiles}
-                    originalFilterRule={originalFilterRule}
-                    availableTileFilters={filterableFieldsByTileUuid}
-                    defaultFilterRule={defaultFilterRule}
-                    onSave={handelSaveChanges}
-                    // FIXME: remove this once we migrate off of Blueprint
-                    popoverProps={{
-                        onOpened: () => openSubPopover(),
-                        onOpening: () => openSubPopover(),
-                        onClose: () => closeSubPopover(),
-                        onClosing: () => closeSubPopover(),
-                    }}
-                />
+                {filterableFieldsByTileUuid && (
```

Comment:
This should probably be !isPopoverDisabled. It should work the same, but it'll be less confusing to keep it symmetric like that.